src/screens/repo/screens/secrets/components/form.js

```diff
@@ -59,6 +90,47 @@ export class Form extends Component {
 					placeholder="Secret Value"
 					onChange={this._handleValueChange}
 				/>
+				<section>
+					<h2>Events</h2>
+					<div>
+						<label>
+							<input
+								type="checkbox"
```

Comment:
Create a list of event / label than iterate over it instead of create list of markup.